airflow/utils/db.py

```diff
@@ -1633,8 +1641,9 @@ def resetdb(session: Session = NEW_SESSION, skip_init: bool = False):
     connection = settings.engine.connect()
 
     with create_global_lock(session=session, lock=DBLocks.MIGRATIONS):
-        drop_airflow_models(connection)
-        drop_airflow_moved_tables(session)
+        with connection.begin():
```

Comment:
Is this needed? The calls here are mostly DDL and I’m not sure a transaction changes things much (and they weren’t wrapped in one previously).

Reply:
Yes. It's needed. The error says:
```log
/opt/airflow/airflow/utils/db.py:1724 RemovedIn20Warning: The current statement is being autocommitted using implicit autocommit, which will be removed in SQLAlchemy 2.0. Use the .begin() method of Engine or Connection in order to use an explicit transaction for DML and DDL statements. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
```